metadata.go

```diff
@@ -826,19 +912,31 @@ func getImageMetadata(tagSlice []TagInfo, oldMetadataSet MetadataSet) (metadataS
 		go func(imageID ImageIDType, ch chan ImageMetadataInfo, errch chan error) {
 			var metadata ImageMetadataInfo
 			blog.Info("Get Metadata for Image: %s", string(imageID))
-			if *RegistryProto == "quay" {
-				// TODO: Properly support quay.io image metadata instead of faking it.
-				t := time.Date(2011, time.January, 1, 1, 0, 0, 0, time.UTC)
-				metadata.Image = string(imageID)
-				metadata.Datetime = t
-				ch <- metadata
-				return
+
+			var response []byte
+			var e error
+
+			if LocalHost {
+				response, e = inspectImage(string(imageID))
+				if e != nil {
+					blog.Info(string(response))
+				}
+			} else {
+				if *RegistryProto == "quay" {
+					// TODO: Properly support quay.io image metadata instead of faking it.
+					t := time.Date(2011, time.January, 1, 1, 0, 0, 0, time.UTC)
+					metadata.Image = string(imageID)
+					metadata.Datetime = t
+					ch <- metadata
+					return
+				}
+				response, e := RegistryQuery(client, RegistryAPIURL+"/v1/images/"+string(imageID)+"/json")
```

Comment:
I'm getting a compile-time error for this line. The variables "response" and "e" are defined in this local scope (the else body) but are not used within that scope.